src/common/roll_renderer.js

```diff
@@ -78,20 +78,25 @@ class Beyond20RollRenderer {
         let advantage = request.advantage;
         if (advantage == RollType.QUERY)
             advantage = await this.queryAdvantage(title);
+        
+        let d20_roll = "1d20";
+        // Halfling Luck affects Attack Rolls
+        if (request.character["racial-traits"].includes("Lucky"))
+            d20_roll = d20_roll.replace(/[0-9]*d[0-9]+/g, "$&ro<=1");
```

Comment:
You've just set it to `"1d20"` so not sure why you need to regexp it, I assume it's a paste from somewhere else ?